yaml file error message incorrectly parsed by summary code

### Version Number

2.2.1

### What branch are you on?

master

### Describe the Bug

```
[ERROR]    | Config Error: No YAML (.yml|.yaml) files found in config/metadata_files                            |
```
in the summary:
```
[INFO]     |     3 | Config Error: No YAML (.yml                                                                |
```

### Relevant Collection/Overlay/Playlist Definition

```yml
metadata_files:
    - folder: config/metadata_files

Where apparently that folder is empty.
```

### Logs

_No response_

--- a/modules/util.py
+++ b/modules/util.py
@@ -381,7 +381,7 @@ def load_files(files_to_load, method, err_type="Config", schedule=None, lib_vars
                     if yml_files:
                         current.extend([("File", yml) for yml in yml_files])
                     else:
-                        logger.error(f"{err_type} Error: No YAML (.yml|.yaml) files found in {file['folder']}")
+                        logger.error(f"{err_type} Error: No YAML (.yml,.yaml) files found in {file['folder']}")
 
             temp_vars = {}
             if "template_variables" in file and file["template_variables"] and isinstance(file["template_variables"], dict):

